scripts/uberenv/packages/axom/package.py

```diff
@@ -162,91 +167,91 @@ def hostconfig(self, spec, prefix):
             if os.path.isfile(env["SPACK_FC"]):
                 f_compiler  = env["SPACK_FC"]
 
-
         # are we on a specific machine
         sys_type = self._get_sys_type(spec)
         on_blueos = 'blueos' in sys_type
         on_blueos_p9 = on_blueos and 'p9' in sys_type
-        on_toss =  'toss_3' in sys_type
+        on_toss = 'toss_3' in sys_type
 
         # cmake
         if "+cmake" in spec:
-            cmake_exe = pjoin(spec['cmake'].prefix.bin,"cmake")
+            cmake_exe = pjoin(spec['cmake'].prefix.bin, "cmake")
         else:
             cmake_exe = which("cmake")
             if cmake_exe is None:
-                #error could not find cmake!
+                # error could not find cmake!
                 crash()
             cmake_exe = cmake_exe.command
 
         host_config_path = self._get_host_config_path(spec)
-        cfg = open(host_config_path,"w")
-        cfg.write("#------------------{}\n".format("-"*60))
+        cfg = open(host_config_path, "w")
+        cfg.write("#------------------{0}\n".format("-" * 60))
         cfg.write("# !!!! This is a generated file, edit at own risk !!!!\n")
-        cfg.write("#------------------{}\n".format("-"*60))
-        cfg.write("# Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and\n")
-        cfg.write("# other Axom Project Developers. See the top-level COPYRIGHT file for details.\n")
-        cfg.write("#\n")
-        cfg.write("# SPDX-License-Identifier: (BSD-3-Clause)\n")
-        cfg.write("#------------------{}\n".format("-"*60))
-        cfg.write("# SYS_TYPE: {}\n".format(sys_type))
-        cfg.write("# Compiler Spec: {}\n".format(spec.compiler))
-        cfg.write("#------------------{}\n".format("-"*60))
+        cfg.write("#------------------{0}\n".format("-" * 60))
```

Comment:
So, we don't want the Axom copyright in the generated host-configs?

Reply:
I don't think we do.  Especially when these are being generated by Spack.